lib/app/components/custom/custom_waveform_icon.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/widgets.dart';
+
+// iOS 17 waveform icon for iMessage
+
+class WaveformIconData extends IconData {
+  const WaveformIconData(int codePoint)
+      : super(
+    codePoint,
+    fontFamily: 'WaveFormIcon',
+  );
+}
```

Comment:
The `CupertinoIcons` class contains a waveform icon that looks pretty close, if not exactly like the iOS one. You can use it via `CupertinoIcons.waveform`, that way we dont need a custom font and whatnot